cli/src/commands/auth/login.ts

```diff
@@ -0,0 +1,41 @@
+import prompts from 'prompts';
+import { BaseCommand } from '../../base.js';
+import { readAPIKeyFromFile, writeAPIKey } from '../../key.js';
+
+export default class Login extends BaseCommand {
+  static description = 'Authenticate with Xata';
+
+  static examples = [];
+
+  static flags = {};
```

Comment:
Add a flag to login without interactivity

Reply:
That'd be handy, but my concern is that the API key will be stored in the command history in the shell.